pkg/rds.go

```diff
@@ -317,6 +317,12 @@ var DBMaxConnections = map[string]map[string]int64{
 	//
 	// R6
 	//
+    "db.r6g.8xlarge": map[string]int64{
+		// Memory: 256 GiB
+		"default":          5000,
+		"default.mysql5.7": 262143,
+		"default.mysql8.0": 262143,
```

Comment:
Could use please add, at the beginning of the map (in the comments), that `5000` is the max value for Postgres on instances with more than 64GB.

Also MySQL is that:

```suggestion
		"default.mysql5.7": 21600,
		"default.mysql8.0": 21600,
```

Reply:
I wasn't aware of this document, so I recently added some more information at the beginning of the mapping. Let me refactor everything in another PR.